ceph/Makefile

```diff
@@ -0,0 +1,21 @@
+all: format vet adapter
```

Comment:
we should not need a build file, let's use KO here

Reply:
removed the makefile